core/chain-configs/src/client_config.rs

```diff
@@ -230,6 +232,7 @@ impl ClientConfig {
             max_gas_burnt_view: None,
             enable_statistics_export: true,
             client_background_migration_threads: 1,
+            flat_storage_creation_period: Duration::from_millis(100),
```

Comment:
I just realized that the `FlatStateMisc` column is not cached (not saying it should, it's shared among different threads so it's probably hard to make it cached), but it means each trigger of `update_status` will hit the disk a few times, so it's good that we set this value to a higher value. I think 100ms is probably ok, but maybe 1s is safer, since I don't think the update_status function needs to be called that often.